style.css

```diff
@@ -40,14 +49,22 @@ header {
   text-decoration: none;
 }
 
+/* Main Section */
+main {
+  width: 95vw;
+  height: auto;
+  background-color: #fff;
+  margin: auto;
+}
+
 /* Headline Section Styling */
 
-section {
+#headline-container {
   background-image: url("./images/header-illsutration-mobile@2x.png");
   background-repeat: no-repeat;
```

Comment:
- [ ] Great job fixing the background image. like the previous reviewer noted, kindly fix the width of the diferent sections so as to avoid the horizontal scroll. 

![demo33](https://user-images.githubusercontent.com/65298048/205398178-4722876f-cd77-486f-a6d5-8372026c088a.gif)